tlsobs-api/handlers.go

```diff
@@ -17,8 +17,10 @@ import (
 	"bytes"
 	"crypto/sha256"
 
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
```

Comment:
Why not put everything into the metrics package and not impact cloudwatch in the api/scanner?